ccd-definition/CaseEventToFields/CareSupervision/manageOrders/5_orderDetailsSection.json

```diff
@@ -310,5 +310,31 @@
     "PageColumnNumber": 1,
     "FieldShowCondition": "manageOrdersEndDateTypeWithMonth=\"CALENDAR_DAY_AND_TIME\" OR manageOrdersEndDateTypeWithEndOfProceedings=\"CALENDAR_DAY_AND_TIME\"",
     "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersUploadOrderFile",
```

Comment:
this new section does not include the is this order final part. Is that a separate story?

Reply:
it's implemented by https://tools.hmcts.net/jira/browse/FPLA-1968 and needs to be merged once in master